tracks/ruby/exercises/clock/mentoring.md

```diff
@@ -252,3 +248,9 @@ read it in a different form. The following pattern:
 is very common in software. Discuss the difference between human representation
 of data and data that is easy to work with.
 
+[Postgres-time-of-day]: https://www.postgresql.org/docs/current/static/datatype-datetime.html
+[UNIX-timestamps]: http://unixtimestamp.50x.eu/about.php
```

Comment:
```suggestion
[UNIX-timestamps]: https://www.epochconverter.com/
```

Reply:
`http://unixtimestamp.50x.eu/about.php`  is throwing a DNS error for me.  I am in the states using ATT, so not sure if it is a provider thing or something else.  Subbed in a different service that does resolve, but we should research/test for one that as equal global access, just to be safe.

Reply:
I hope you like how I resolved it!